storage/integration_test.go

```diff
@@ -3994,6 +3995,69 @@ func TestIntegration_Scopes(t *testing.T) {
 
 }
 
+func TestBucketSignURL(t *testing.T) {
```

Comment:
This is nice-- we can probably just replace existing signed URL integration tests with tests that use the wrapper method.